prboom2/src/m_menu.c

```diff
@@ -3300,25 +3300,25 @@ static const char *gen_skillstrings[] = {
 
 static const char *gen_compstrings[] =
 {
-  "Default",
-  "Doom v1.2",
-  "Doom v1.666",
-  "Doom/2 v1.9",
-  "Ultimate Doom",
-  "Final Doom",
-  "DosDoom",
-  "TASDoom",
-  "Boom's vanilla",
-  "Boom v2.01",
-  "Boom",
-  "LxDoom",
-  "MBF",
-  "PrBoom 2.03b",
-  "PrBoom 2.1.x",
-  "PrBoom 2.2.x",
-  "PrBoom 2.3.x",
-  "PrBoom 2.4.0",
-  "Latest PrBoom+",
+  "17/Default",
+  " 0/Doom v1.2",
+  "  1/Doom v1.666",
+  " 2/Doom/2 v1.9",
+  " 3/Ultimate Doom",
+  " 4/Final Doom",
+  " 5/DosDoom",
+  " 6/TASDoom",
+  " 7/Boom's vanilla",
+  " 8/Boom v2.01",
+  " 9/Boom",
+  "10/LxDoom",
+  " 11/MBF",
```

Comment:
Here as well?

Reply:
It's based on the width of the widest number, hence why everything from 12 onwards looks evenly spaced.

Reply:
The point is, I don't see why these should be aligned at all, because you'll never see more than one at a time.

Reply:
It lines up the names with where they previously were, so that it could fit existing names without shortening them.

Reply:
Sorry, but I am not going to merge it like this. Prboom+'s menu is already a mess, but having single lines in a paragraph aligned differently is definitely not going to make it any better.